fastlane/Fastfile

```diff
@@ -266,8 +266,8 @@ def attach_changelog_to_master(version_number)
   }
 end
 
-def replace_in(previous_text, new_text, path)
-  if new_text.to_s.strip.empty?
+def replace_in(previous_text, new_text, path, allow_empty=false)
+  if new_text.to_s.strip.empty? and not allow_empty
```

Comment:
the recent changes in the fastlane broke this, since we were relying on setting to an empty proxy url if one wasn't present.